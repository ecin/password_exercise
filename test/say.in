a
tv
ptoui
bontres
zoggax
wiinq
eep
houctuh
ei
cd
bcdfghjklmnpqrstvwxy
ee
oo
jj
aei
vwx
to
in
try
ask
bot
abcodefgohijkolmonop
jkloxoxoxoxoxoxoxoxo
mamamamajklamamamama
egegegegegegegegejkl
aeiququququququququq
orororaeirororororor
zezezezezezezezezoua
eeprop
peerop
propee
ooplant
poolant
plantoo
ssuper
supper
superr
aarmada
armaada
armadaa
banana
rhythm
breakneck
