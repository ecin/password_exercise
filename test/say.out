<a> is acceptable.
<tv> is not acceptable.
<ptoui> is not acceptable.
<bontres> is not acceptable.
<zoggax> is not acceptable.
<wiinq> is not acceptable.
<eep> is acceptable.
<houctuh> is acceptable.
<ei> is acceptable.
<cd> is not acceptable.
<bcdfghjklmnpqrstvwxy> is not acceptable.
<ee> is acceptable.
<oo> is acceptable.
<jj> is not acceptable.
<aei> is not acceptable.
<vwx> is not acceptable.
<to> is acceptable.
<in> is acceptable.
<try> is not acceptable.
<ask> is acceptable.
<bot> is acceptable.
<abcodefgohijkolmonop> is acceptable.
<jkloxoxoxoxoxoxoxoxo> is not acceptable.
<mamamamajklamamamama> is not acceptable.
<egegegegegegegegejkl> is not acceptable.
<aeiququququququququq> is not acceptable.
<orororaeirororororor> is not acceptable.
<zezezezezezezezezoua> is not acceptable.
<eeprop> is acceptable.
<peerop> is acceptable.
<propee> is acceptable.
<ooplant> is acceptable.
<poolant> is acceptable.
<plantoo> is acceptable.
<ssuper> is not acceptable.
<supper> is not acceptable.
<superr> is not acceptable.
<aarmada> is not acceptable.
<armaada> is not acceptable.
<armadaa> is not acceptable.
<banana> is acceptable.
<rhythm> is not acceptable.
<breakneck> is acceptable.
